public/app/features/trails/DataTrail.tsx

```diff
@@ -137,8 +137,8 @@ export class DataTrail extends SceneObjectBase<DataTrailState> {
       );
     }
 
-    // Save the current trail as a recent if the browser closes or reloads
-    const saveRecentTrail = () => getTrailStore().setRecentTrail(this);
+    // Save the current trail (if it has a metric) as a recent if the browser closes or reloads
+    const saveRecentTrail = () => this.state.metric && getTrailStore().setRecentTrail(this);
```

Comment:
Check for those filters with this special Scenes variable lookup 👀 

```suggestion
    const saveRecentTrail = () => {
      const filtersVariable = sceneGraph.lookupVariable(VAR_FILTERS, this);
      const hasFilters = filtersVariable instanceof AdHocFiltersVariable && filtersVariable.state.filters.length > 0;

      return ((this.state.metric || hasFilters) 
      && getTrailStore().setRecentTrail(this))};```

Reply:
The variable lookup is interesting 🤔 You might think the filters are stored in the state but this `VAR_FILTERS` variable is one way that Scenes can access stored data across all the Scenes components.